### DISCUSSION TITLE
Has something changed ?

### DISCUSSION THREAD

**Discussion by Gui-Yom** [emilk/egui#318] [Q&A] [answered]
I have the following UI code (trimmed down so straight to the point) :
```rust
self.egui_platform.begin_frame();

        egui::SidePanel::left("params", self.settings.ui_width as f32).show(
            &self.egui_platform.context(),
            |ui| {
                ui.label("Params");
                for param in self.params.iter_mut() {
                    ui.add(
                        DragValue::f32(&mut param.value)
                            .prefix(format!("{}: ", param.name))
                            .clamp_range(param.min..=param.max)
                            .max_decimals(3)
                            .speed(
                                param.max
                                    / (window_size.width as f32
                                        - self.settings.ui_width as f32 * scale_factor),
                            ),
                    );
                }
            },
        );
        egui::CentralPanel::default().frame(Frame::none()).show(
            &self.egui_platform.context(),
            |ui| {
                ui.image(
                    TextureId::User(0),
                    egui::Vec2::new(
                        (window_size.width as f32 - self.settings.ui_width as f32 * scale_factor)
                            / scale_factor,
                        window_size.height as f32 / scale_factor,
                    ),
                );
            },
        );
```
Which gives this with egui 0.10 : 
![ui_trimmed_old](https://user-images.githubusercontent.com/25181283/115151844-f60fa800-a06e-11eb-8279-dab3196519a5.png)

After updating to egui 0.11, I get this : 
![ui_trimmed_new](https://user-images.githubusercontent.com/25181283/115151843-f60fa800-a06e-11eb-8565-4e243526bcfb.png)

Here are the versions with everything : 
Old
![ui_old](https://user-images.githubusercontent.com/25181283/115151842-f5771180-a06e-11eb-837e-525768bab4cd.png)
New
![ui_new](https://user-images.githubusercontent.com/25181283/115151841-f4de7b00-a06e-11eb-80a2-2c725a6a5aad.png)

It seems the first DragValue is destroying the width for every widget after. Has anything changed with egui 0.11 that I'm not aware of ? Any pointers ? Is this a bug ?

**Comment by emilk** [comment, marked as answer]
This is a bug that sneaked in - thanks for noticing! Push fixed in https://github.com/emilk/egui/commit/035166276322b3f2324bd8b97ffcedc63fa8419f

**Reply by ls-ivan-kiselev** [reply]
Seems to have similar bug in version `0.20.1`:
The code:
```rust
        tracing::info!("before: {:?}", ui.available_size_before_wrap());
        ui.add_sized(left_size, |ui: &mut Ui| ui.label(age));
        tracing::info!("after: {:?}", ui.available_size_before_wrap());
```
The output:

```
before: [472.0 100.0]
after: [0.0 100.0]
```
